.github/workflows/publish-stable.yml

```diff
@@ -48,6 +48,16 @@ jobs:
       tag: ${{ steps.get_tag.outputs.tag }}
       prev_tag: ${{ steps.get_prev_tag.outputs.prev_tag }}
 
+  comment:
+    name: Comment on PRs included in release
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    needs: tag
+    if: needs.tag.outputs.is_stable == 'true'
+    steps:
+      - uses: nflaig/release-comment-on-pr@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@wemeetagain as noted in the [usage section](https://github.com/marketplace/actions/comment-on-pull-requests-included-in-release#usage) of the gh action, this token needs the `public_repo` permission which is not the case by default. Might need to create a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) in that case and use that one instead.

![image](https://user-images.githubusercontent.com/38436224/223111599-c9607e14-c4be-4976-932e-8c106b7d70ce.png)